DisplaySettings/DisplaySettings.cpp

```diff
@@ -4390,5 +4453,75 @@ namespace WPEFramework {
             }
             returnResponse(success);
         }
+        uint32_t DisplaySettings::getVideoFormat(const JsonObject& parameters, JsonObject& response)
+        {   //sample servicemanager response:{"standards":["HDR10"],"supportsHDR":true}
+            LOGINFOMETHOD();
+
+            JsonArray videoFormats;
+            int capabilities = dsHDRSTANDARD_NONE;
+
+            try
+            {
+                device::VideoDevice &device = device::Host::getInstance().getVideoDevices().at(0);
+                device.getHDRCapabilities(&capabilities);
+            }
+            catch(const device::Exception& err)
+            {
+                LOG_DEVICE_EXCEPTION0();
+            }
+
+            videoFormats.Add("SDR");
+            if(capabilities & dsHDRSTANDARD_HDR10)videoFormats.Add("HDR10");
+            if(capabilities & dsHDRSTANDARD_HLG)videoFormats.Add("HLG");
+            if(capabilities & dsHDRSTANDARD_DolbyVision)videoFormats.Add("DV");
+            if(capabilities & dsHDRSTANDARD_TechnicolorPrime)videoFormats.Add("Technicolor Prime");
+            try
+            {
+                std::string strVideoPort = device::Host::getInstance().getDefaultVideoPortName();
+                device::VideoOutputPort vPort = device::Host::getInstance().getVideoOutputPort(strVideoPort.c_str());
+                if (vPort.isDisplayConnected())
+                {
+                    int _eotf = vPort.getVideoEOTF();
+                    switch (_eotf)
```

Comment:
Once you have video format......code is common for both notification and get request so create a function to convert into responseParam

Reply:
Refactoring done